### PR TITLE
CASMINST-3495: Update info about troubleshooting Prometheus alerts

### PR DESCRIPTION
**Pull request by haroldlongley**
## Summary and Scope

CASMINST-3495: Update info about troubleshooting Prometheus alerts

## Testing

Documentation only change.

### CODE REVIEW

File: operations/system_management_health/Access_System_Management_Health_Services.md [hunk 1]
```diff
@@ -32,12 +32,16 @@ This procedure enables administrators to set up the service and access its compo
 
         For more information regarding the use of the Prometheus interface, see [https://prometheus.io/docs/prometheus/latest/getting\_started/](https://prometheus.io/docs/prometheus/latest/getting_started/).
 
+        Some alerts may be falsely triggering whether based on alerts which will be improved in the future or alerts related to whether all software products have been installed yet. See [Troubleshoot Prometheus Alerts](operations/system_management_health/Troubleshoot_Prometheus_Alerts.md)
+
     -   **https://alertmanager.\{\{shasta\_domain\}\}/**
 
         Central Alertmanager instance that manages prometheus alerts.
```
Comment (nrockershousen) [inline operations/system_management_health/Access_System_Management_Health_Services.md:39]: prometheus -> Prometheus

File: operations/system_management_health/Troubleshoot_Prometheus_Alerts.md [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
 - [PostgresqlInactiveReplicationSlot](#inactiveslot)
 - [PostgresqlNotEnoughConnections](#notenoughconnections)
-- [CPUThrottlingHigh](#cputhrottlinghigh)
+
+<a name="cputhrottlinghigh"></a>
+## CPUThrottlingHigh
+
+Alerts for CPUThrottlingHigh on gatekeeper-audit can be ignored. This pod is not utilized in this release.
+
+Alerts for CPUThrottlingHigh on CFS services such as cfs-batcher and cfs-trust can be ignored. Because CFS is idle most of the time these services have low CPU requests, and it is normal for CFS service resource usage to spike when it is in use.
```
Comment (nrockershousen) [inline operations/system_management_health/Troubleshoot_Prometheus_Alerts.md:17]: expand first use of CFS -> Configuration Framework Service (CFS)

File: operations/system_management_health/Access_System_Management_Health_Services.md [hunk 3]
```diff
@@ -32,12 +32,16 @@ This procedure enables administrators to set up the service and access its compo
 
         For more information regarding the use of the Prometheus interface, see [https://prometheus.io/docs/prometheus/latest/getting\_started/](https://prometheus.io/docs/prometheus/latest/getting_started/).
 
+        Some alerts may be falsely triggering whether based on alerts which will be improved in the future or alerts related to whether all software products have been installed yet. See [Troubleshoot Prometheus Alerts](operations/system_management_health/Troubleshoot_Prometheus_Alerts.md)
```
Comment (nrockershousen) [inline operations/system_management_health/Access_System_Management_Health_Services.md:35]: - Missing period in second line.
- Changing the wording to something similar to the following might make this easier to follow:

Some alerts may be falsely triggered. This occurs if they are alerts which will be improved in a future release, or if they are alerts impacted by if all software products have been installed yet.

File: operations/system_management_health/Access_System_Management_Health_Services.md [hunk 4]
```diff
@@ -32,12 +32,16 @@ This procedure enables administrators to set up the service and access its compo
...
+
     -   **https://alertmanager.\{\{shasta\_domain\}\}/**
 
         Central Alertmanager instance that manages prometheus alerts.
 
         The alertmanager manages the alerts it receives and generates notifications to users or applications. For more information about `alert-manager`, refer to the following documentation: [https://prometheus.io/docs/prometheus/latest/getting\_started/](https://prometheus.io/docs/prometheus/latest/getting_started/).
 
+        Some alerts may be falsely triggering whether based on alerts which will be improved in the future or alerts related to whether all software products have been installed yet. See [Troubleshoot Prometheus Alerts](operations/system_management_health/Troubleshoot_Prometheus_Alerts.md)
```
Comment (nrockershousen) [inline operations/system_management_health/Access_System_Management_Health_Services.md:43]: same comments as above

**Review by nrockershousen — APPROVED** [review]

**Comment by haroldlongley** [comment]
Closing this and fixing the problems in a different branch for release/1.0.

**Pull request closed without merging**